config/runtime_flags.go

```diff
@@ -3,20 +3,22 @@ package config
 import "maps"
 
 var (
+	// RuntimeFlagConfigHotReload enables the hot-reloading mechanism for the config file
+	// and any other files referenced within it
+	RuntimeFlagConfigHotReload = runtimeFlag("config_hot_reload", true)
+
+	// RuntimeFlagEnvoyResourceManager enables Envoy overload settings based on
+	// process cgroup limits (Linux only).
```

Comment:
@kralicky please let me know if this description is accurate, or if there's anything else you'd want to add here. Thanks!

Reply:
This LGTM